UPGRADING.md

```diff
@@ -1,6 +1,20 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.9.0
+
+#### Response Headers
+
+For Rack >= 3 the following response headers are returned lowercase:
```

Comment:
This should say something like "As per https://github.com/rack/rack/issues/1592 Rack 3.0 is enoforcing the HTTP/2 semantics, and thus treats all headers as lowercase. Starting with Grape 1.9.0, the following headers are now lowercase: ..."